lib/ansible/plugins/inventory/docker_machine.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Ximon Eighteen <ximon.eighteen@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: docker_machine
+    plugin_type: inventory
+    short_description: Docker Machine inventory source
+    requirements:
+        - L(Docker Machine,https://docs.docker.com/machine/)
+    extends_documentation_fragment:
+        - inventory_cache
+        - constructed
+    description:
+        - Get inventory hosts from Docker Machine.
+        - Uses a YAML configuration file that ends with docker_machine.(yml|yaml).
+        - The plugin returns an I(all) group of nodes and one group per driver (e.g. digitalocean).
+        - The plugin sets standard host variables I(ansible_host), I(ansible_port), I(ansible_user) and I(ansible_ssh_private_key).
+        - The plugin also sets standard host variable I(ansible_ssh_common_args) to '-o StrictHostKeyChecking=no'.
```

Comment:
```suggestion
        - The plugin also sets standard host variable I(ansible_ssh_common_args) to C(-o StrictHostKeyChecking=no).
```
C(...) is also commonly used for general "code"/"teletype" markings

Reply:
I based that style on the `docker_swarm` inventory plugin documentation. Is it desirable for the two plugins to use different approaches to styling their documentation?

Reply:
All modules and plugins should use the same style; the problem is that not all of them do so (yet). The `docker_swarm` inventory plugin is pretty new and might not yet be perfect (while looking over it right now, I found a few problematic ones and just created a PR for that: #54952).